portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,255 @@
 <!DOCTYPE html>
-<html>
+<html lang="en" dir="ltr">
   <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link rel="stylesheet" href="css/animate.css">  
+    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="wrapper">
+        <header>
+            <div class="container">
+                <div class="logo">
+                    <h1>P E N I E L  &nbsp;  A B E B E</h1>
+                </div>
+                <nav>
+                    <ul>
+                        <li><a href="#about">about me</a></li>
+                        <li><a href="#skills">skills</a></li>
+                        <li><a href="#projects">projects</a></li>
+                        <li><a href="#gallery">gallery</a></li>
+                        <li><a href="#contact">contact</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
     </div>
+    <section>
+        <div class="greeting-box">
+            <h1>Hi There, Welcome to My Portfolio</h1>
+            <h3>click here to learn random facts about me</h3>
+            <button onclick="addRandomGreeting()">Hey</button>
+            <div id="greeting-container"></div>
+        </div>
+    </section>
+    <section class="about" id="about">
+        <div class="container">
+            <div class="profile-img">
+                <img src="images/profile.jpg" alt="">
+            </div>
+            <div class="about-details">
+                <div class="about-heading">
+                    <h1>About</h1>
+                    <h6>Myself</h6>
+                </div>
+                <p>
+                    My name is Peniel Abebe. I am a rising Senior, Computer Science major at Howard University from Ethiopia.
+                    I currently live in Maryland but I call many places home as I've moved around a lot.
+                    I am passionated about Computer Science because it is a very powerful tool that has made our world more connected, faster and created solutions to so many of our problems.
```

Comment:
Style suggestion: Please wrap long lines at some reasonable width. Lines longer than 150 chars don't fit into github diff view making it inconvenient tot read the content.

Reply:
Makes sense! is it better to resolve that by adding the "word-wrap: break-word;" property on the css or add more <p> tags in between the paragraphs?

Reply:
What I meant was to avoid long lines in html source code. A single long line will be rendered in HTML exactly the same way as the same text broken into multiple lines (like the following two fragments). The disadvantage of single long line is that it requires horizontal scrolling when reading the code:
```
I am passionated about Computer Science because it is a very powerful tool that has made our world more connected, faster and created solutions to so many of our problems.
```
---
```
I am passionated about Computer Science
because it is a very powerful tool that has
made our world more connected, faster and
created solutions to so many of our problems.
```

In general line length of the source code is typically present in style guides for most languages: https://google.github.io/styleguide/javaguide.html#s4.4-column-limit
HTML doesn't have strict line wrapping limit or rule. It is more like recommendation: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

Reply:
Oh, I see. Okay will keep that in mind